packages/ui/atomic/create-atomic-component/template/src/components/sample-component/src/sample-component.tsx

```diff
@@ -51,6 +51,8 @@ export class SampleComponent {
   // Using `componentWillLoad` will hang the parent atomic-search-interface initialization.
   public async connectedCallback() {
     try {
+      // Wait for the Atomic to load.
+      await customElements.whenDefined('atomic-search-interface');
```

Comment:
I think there is already a method for that in `packages/ui/atomic/template/src/utils/atomic.ts` called
```
waitForAtomic()
```

Reply:
Yep, it's a copy from there, but its in another package, so that's where the copy come from